browser-extension/manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "Force \"Latests Tweets\" on first load",
```

Can we keep the same wording and punctuation for consistency?

```suggestion
  "name": "Fuck off, @twitter",
```